system/Files/FileSizeUnit.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Files;
+
+use InvalidArgumentException;
```

Comment:
Almost forgot about this - in 4.6 we now use framework exceptions, if available.
```suggestion
use CodeIgniter\Exceptions\InvalidArgumentException;
```